OrbitGl/CaptureWindow.cpp

```diff
@@ -1030,6 +1030,13 @@ void CaptureWindow::RenderToolbars() {
   current_x += ImGui::GetWindowWidth() + space_between_toolbars;
   ImGui::End();
 
+  // Set function filter text programmatically if needed.
+  bool new_filter = Capture::GFunctionFilter != find_filter_;
+  if( new_filter ) {
+    strncpy(find_filter_, Capture::GFunctionFilter.c_str(), kFilterLength);
```

Comment:
is there a reason for find_filter to be char[] instead of std::string?

Reply:
Yes, ImGui expects a buffer of chars of known size into which it will copy the user input.  Using a string could work but the string would be ill formed as its size wouldn't be updated by ImGui.

Reply:
maybe we should all one to OrbitBase.

Reply:
good idea.